### ISSUE TITLE
running code on colab

### ISSUE THREAD

**Issue by huda1197** [HSE-asavchenko/face-emotion-recognition#34]
Hello,
I want to run the code on google colab on the pre-trained models, 
after reading the read.me I'm trying to run display_emotions.ipynb but I'm getting errors.
Is it right to start working on display_emotions.ipynb?

**Comment by HSE-asavchenko** [comment]
Hello! It should work on colab, but probably some packages need to be installed or the correct version is required. Complete list of packages is available in [requirements](https://github.com/HSE-asavchenko/face-emotion-recognition/blob/main/src/requirements.txt) but not all of them are required to run display_emotions.
Please, copy the stack trace for your errors.

If you just want to test the models, you could start with much simpler [example](https://github.com/HSE-asavchenko/hsemotion/blob/main/demo/test_hsemotion_package.ipynb) of my auxiliary hsemotion repository

**Comment by huda1197** [comment]
thanks for your answers.
actually I want to run it then retrain it on affectnet data set. I can do that without getting the other datasets you used in your code right?
I'm facing two errors the first one is when I try to excute the block {from facial_analysis import FacialImageProcessing
imgProcessing=FacialImageProcessing(False)} 
it gives ModuleNotFoundError: No module named 'facial_analysis'
though I executed facial_analysis in colab using !python command

the other error happened when I copied all the content of facial_analysis in cells in display_emotions it all run perfectly but when i try to call the FacialImageProcessing class it gives error that name '__file__' is not defined
like in the picture.
![Screenshot (759)](https://user-images.githubusercontent.com/82894112/235917815-4c6aad9f-7923-41b5-8777-67e9f1371403.png)

note that I changed the _file_ and replaced it with the path for the file in google drive and it gave me no such file or directory.
